doc/getting_started/installation_mac.rst

```diff
@@ -55,10 +55,17 @@ the brew command line.
 
 .. code-block:: console
 
-   $ brew install gettext qemu help2man mpfr gmp coreutils wget python3 dfu-util
+   $ brew install gettext qemu help2man mpfr gmp coreutils wget python3 dfu-util dtc
```

Comment:
What I feel is missing here is a split between tools and toolchain. For ARM, it is much easier to use GNU ARM Embedded and only install qemu and dtc, so ideally it would be great if we could separate those in 2 different steps